docs/Ace.md

```diff
@@ -53,6 +54,19 @@ virtual int run( int checkpoint ) override {
 }
 ```
 
+2. *XFSTest* : The XFSTest adapter translates the high-level language into a test file and expected output file to be run with [xfstest](https://github.com/kdave/xfstests). For example, the above workload would be converted into the following code (excluding the xfstest initializiation and code and helper methods):
```

Comment:
Could you add the commands to generate files using this adapter in the readme?
We can merge it in once that's done